src/work-allocation/containers/available-tasks/available-tasks.component.html

```diff
@@ -1,6 +1,6 @@
 <exui-available-tasks-filter (selectionChanged)="onLocationsChanged($event)"></exui-available-tasks-filter>
 <exui-task-list
   [taskServiceConfig]="taskServiceConfig" [tasks]="tasks" [fields]="fields"
-  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage"
+  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage" [sourceColumn]="sourceColumn" [matchValue]="matchValue"
   (sortEvent)="onSortHandler($event)" (actionEvent)="onActionHandler($event)">
 </exui-task-list>
```

Comment:
Linting will fail without a line break at the end.

Reply:
Added